openfecwebapp/config.py

```diff
@@ -12,7 +12,8 @@
 api_key_public = env.get_credential('FEC_WEB_API_KEY_PUBLIC', '')
 cache = os.getenv('FEC_WEB_CACHE')
 cache_size = int(os.getenv('FEC_WEB_CACHE_SIZE', 1000))
-
+transition_url = env.get_credential('FEC_TRANSITION_URL', 'http://www.fec.gov')
```

Comment:
I just changed it to default to `transition.fec.gov` but still have the possibility of overwriting with an env var just in case we need to change it suddenly.